react/features/invite/reducer.js

```diff
@@ -29,6 +29,7 @@ ReducerRegistry.register('features/invite', (state = DEFAULT_STATE, action) => {
         const { numbers, numbersEnabled } = action.dialInNumbers;
 
         return {
+            ...state,
```

Comment:
This fixed an issue where the info dialog could be closed automatically because the value of infoDialogVisible would get blown away.